Public/Src/Cache/ContentStore/Vsts/BackingContentStore.cs

```diff
@@ -39,6 +39,8 @@ public enum SessionCounters
         private readonly IAbsFileSystem _fileSystem;
         private readonly IArtifactHttpClientFactory _artifactHttpClientFactory;
         private readonly TimeSpan _timeToKeepContent;
+        private readonly TimeSpan _pinInlineThreshold;

```

Comment:
Comments here to explain how all this should work and what role each parameter plays.